backend/tessend/data.go

```diff
@@ -0,0 +1,100 @@
+package tessend
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	v2 "github.com/sensu/sensu-go/api/core/v2"
+)
+
+const (
+	// TimestampFormat is the string representation for the license timestamps e.g.
+	// "2018-07-26T12:12:06-04:00"
+	TimestampFormat = time.RFC3339
+)
+
+// Data is the payload sent to tessen
+type Data struct {
+	// General information about the Sensu installation.
+	Cluster Cluster `json:"cluster"`
+
+	// Metric data.
+	Metrics v2.Metrics `json:"metrics"`
+}
+
+// Cluster is the cluster information sent to tessen
+type Cluster struct {
+	// ID is the ID of the sensu-enterprise-go cluster.
+	ID string `json:"id"`
+
+	// Version is the Sensu release version (e.g. "1.4.1").
+	Version string `json:"sensu_version"`
+
+	// License is the Cluster's license.
+	License License
```

Comment:
```suggestion
	License License `json:"license"`
```

Reply:
This should be updated in https://github.com/sensu/tessen/blob/8dda7b83cd6318f124ce76a7e948249f622501cf/v2/types/data.go

Reply:
@echlebek https://github.com/sensu/tessen/pull/10